src/main/scala/components/memory/memory-noncombin-ports.scala

```diff
@@ -30,6 +41,17 @@ class INonCombinMemPort extends ICombinMemPort {
  * The I/O for this module is defined in [[DMemPortIO]].
  */
 class DNonCombinMemPort extends BaseDMemPort {
+  // To fake combinational writes
+  val dmemBusy  = Reg (new Bool)
+  dmemBusy := DontCare
+
+  io.pipeline.good := (! dmemBusy || io.bus.response.valid)
```

Comment:
Same as above. Also, let's stick with `!` instead of `~` (see above). This is a *logical* statement. By using bitwise invert, if the size of dmemBusy changes a bug could be introduced.